index.html

```diff
@@ -424,36 +424,123 @@ <h4>Double-sided ruby, when both read aloud</h4>
+
+	  <section>
+	    <h5>Incorrectly pronouncing non-particle は or へ as particles</h5>
+
+	    <p>In modern Japanese, there is basically only one way to read
+	      each hiragana character. But <span lang="ja">は</span> and
+	      <span lang="ja">へ</span> are exceptions. <span lang="ja">は</span>
+	      is usually read aloud as /ha/ but is read aloud as /wa/ when
+	      it is used as a particle. Likewise, <span lang="ja">へ</span>
+	      is usually read aloud as /he/ but is read aloud as /e/ when
+	      it is used as a particle.</p>
+
+	    <p>Reading aloud ruby annotations rather than ruby bases implies
+	      that CJK ideographic characters in ruby bases will not be
+	      passed to the TTS engine, only hiragana characters in ruby
+	      annotations will be.</p>
+
+	    <p>Without CJK ideographic characters, Japanese morphological
+	      analysis is likely to fail. Thus, occurrences of
+	      <span lang="ja">は</span> or <span lang="ja">へ</span> as
+	      non-particles in ruby annotations may well be mistakenly
+	      interpreted as particles.  Consequently, such occurrences
+	      of <span lang="ja">は</span> and <span lang="ja">へ</span>
+	      may well be mistakenly read aloud as /wa/ and /e/, respectively.</p>
+
+	    <p>For example, consider  
+	      <span lang="ja">やがて<ruby><rb>廃止</rb><rt>はいし</rt></ruby>になる</span>.
+	      This sentence means "It will be abolished eventually".  But
+	      if <span lang="ja">やがてはいしになる</span> is passed to the
+	      TTS engine, <span lang="ja">は</span> may well be mistakenly
+	      read aloud as /wa/ rather than /ha/.  The result means
+	      "I will eventually become a doctor".
+	    </p>
+
+	    <p>Here are some similar examples.  All occurrences of
+	      <span lang="ja">は</span> and <span lang="ja">へ</span>
+	      in ruby annotations are likely to be mistakenly read aloud.</p>
+	    
+	    <ul>
+	      <li><span lang="ja">人員<ruby><rb>配置</rb><rt>はいち</rt></ruby></span> </li>
+	      <li><span lang="ja">自然<ruby><rb>破壊</rb><rt>はかい</rt></ruby></span> </li>
+	      <li><span lang="ja">社会<ruby><rb>波紋</rb><rt>はもん</rt></ruby></span> </li>
+	      <li><span lang="ja">天皇<ruby><rb>陛下</rb><rt>へいか</rt></ruby></span></li>
+	      <li><span lang="ja">大学<ruby><rb>併願</rb><rt>へいがん</ruby></span></li>
+	      <li><span lang="ja">学級<ruby><rb>閉鎖</rb><rt>へいさ</rt></ruby></span></li>
+	    </ul>
+	  </section>
+	  <section>
+	    <h5>Inconsistency between the first and subsequent occurrences</h5>
+            <p>
+	      As described in <a href="#furigana-background"><span class="secno">2.1</span></a>,
+	      furigana as a ruby annotation may be attached to only the
+	      first occurrence of a CJK ideographic character or a word
+	      composed from such characters.  Thus, there is a risk that
+	      the first occurrence and the others are read aloud differently.
+	      For example, consider <span lang="ja">智子</span> as the name
+	      of a character in the novel. There are two possible readings
```

Comment:
better to be wrote as `there are several possible readings of this name, such as ...` (at least three exist for this...)

Reply:
Sure, I can add possible outcomes.